homeassistant/components/fritz/const.py

```diff
@@ -49,6 +50,15 @@ class MeshRoles(StrEnum):
 SERVICE_REBOOT = "reboot"
 SERVICE_RECONNECT = "reconnect"
 SERVICE_CLEANUP = "cleanup"
+SERVICE_SET_GUEST_WIFI_PW = "set_guest_wifi_password"
+
+SERVICE_SCHEMA_SET_GUEST_WIFI_PW = vol.Schema(
+    {
+        vol.Required("device_id"): str,
+        vol.Optional("password"): vol.Length(min=8, max=63),
+        vol.Optional("length"): vol.Range(min=8, max=63),
+    }
+)
```

Comment:
The schema definition should be moved into the `services.py` since it it is only used there